GoogleForCreators/web-stories-wp
# Dashboard: Updated Cooking template to use shapes instead of images when possible.

## Summary

This PR updates the Cooking template to use our new fancy SVG grids and arrows in place of the images (for arrows) and tons of line rectangles (for the grid) we had before.

The template should render like it did before, it's just more efficiently made.

## User-facing changes

- Should be a transparent change, Cooking template should render as it did before.

## Testing Instructions

1.) Go to "Explore Templates".
2.) Click to "View" the Cooking template.
3.) Ensure each page renders like the screenshots below.

## Screenshots

<img src="https://user-images.githubusercontent.com/40646372/86282017-4aef3a00-bb93-11ea-8a77-e955d9ab94f6.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/86282055-5cd0dd00-bb93-11ea-9794-8caf79abaff2.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/86282080-65c1ae80-bb93-11ea-812b-59d26992e93e.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/86282093-6d815300-bb93-11ea-85c5-d48a382646dd.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/86282107-740fca80-bb93-11ea-8f92-445af935ddfa.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/86282592-6870d380-bb94-11ea-8d02-797e59f54dc8.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/86282145-84c04080-bb93-11ea-8dc2-fe9b7a2419f5.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/86282159-8e49a880-bb93-11ea-96b0-c6cb0320bb35.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/86282941-04024400-bb95-11ea-97a3-e6109ae302de.png" height="200">
